modules/ROOT/pages/mongodb/mongodb-connector-6-0-upgrade-migrate.adoc

```diff
@@ -0,0 +1,328 @@
+* String fileName
+* Long Length
+* Int ChunkSize
+* Datetime uploadDate
+* JSON Metadata
+|===
+
+== Removed Operations
+
+The following operations were removed from the MongoDB connector:
+
+[%header,cols="15%,35%,15%,35%"]
+|===
+2+|Removed from MongoDB 5.x 2+| Can be reproduced in MongoDB 6.0.0 through
+|Name	| Description	|Name	|Description
+|Incremental dump | Executes an incremental dump of the database. | Dump | Use the Dump operation.
+|Restore | Takes the output from the dump and restores it | Restore from file or Restore from directory | These operations take the output from the dump file or directory and restore it.
+|Update documents by function | Update documents using a Mongo function | N/A | N/A
+|Update documents by functions | Update documents using one or more Mongo functions | N/A | N/A
+|Find one and update document | Finds and updates the first document that matches a given query | N/A | This operation's functionality can be reproduced by combining other operations
+|Save document | Inserts or updates a document based on its object ID | N/A | N/A
+|Find one document | Finds the first document that matches a given query | Find documents | Set the Limit=1 to return one document.
+|Execute generic command | Executes a generic command on the database | Execute command | Executes a command on the database
+|List files | Lists all files that match the given query and sorts them by filename | N/A | N/A
+|Find one file | Returns the first file that matches the given query | Find files | Giving a specific file to the query
+|===
+
+== Changes in Input Sources
+
+MongoDB 6.0.0 has one input source, Object Listener, which retrieves all of the created documents that belong to a specific collection.
+
+The Delete Sources and Update Sources input sources were removed.
+
+=== Metadata in Object Listener
+
+[%header,cols="50%,50%"]
+|===
+|Input Metadata	|Output Metadata
+
+| N/A
+| The connector resolves output metadata dynamically based on the selected value of the Collection Name parameter. The connector reads the last document of the given collection and uses the document's structure as output metadata.
+|===
+
+== Requirements and Limitations
+
+== Upgrade Prerequisites
+
+Before you perform the upgrade, you must:
+
+. Create a backup of your files, data, and configuration in case you need to restore to the previous version.
+.Install MongoDB v6.0 to replace the MongoDB operations that were previously included in MongoDB Connector v5.x.
+
+== Upgrade Steps
+
+Follow these steps to perform the upgrade from MongoDB Connector v5.3.x to MongoDB Connector v6.0:
+
+. In Anypoint Studio, create a Mule project.
+. In the Mule Palette view, click Search in Exchange.
+. In the *Add Dependencies to Project* window, enter `MongoDB in the search field.
+. In the *Available modules* section, select *MongoDB Connector* and click *Add*.
+. Click *Finish*.
+. Verify that the salesforce-connector dependency version is 6.0.0 in the `pom.xml` file.
```

Comment:
mongodb connector

Reply:
Do you mean the 'pom.xml' file for the MongoDB Connector?

Reply:
the pom.xml file in the Mule Project